dockerfiles/theia/Dockerfile

```diff
@@ -96,6 +136,8 @@ ENV USE_LOCAL_GIT=true \
 
 EXPOSE 3100 3130
 
+COPY --from=plugins-builder /home/theia-dev/che-theia-plugins /default-theia-plugins
```

Comment:
`--chown=theia:root` ?

Reply:
permissions are updated afterwards for `/default-theia-plugins`
and group or user doesn't exist yet.